src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/LanguageServices/Workspace.cs

```diff
@@ -525,16 +526,16 @@ async Task ApplyInBatchAsync()
                     isActiveEditorContext: _activeEditorContextTracker.IsActiveEditorContext(ContextId),
                     isActiveConfiguration: IsPrimary);
 
-                Context.StartBatch();
-
                 try
                 {
+                    IAsyncDisposable disposableBatchScope = await Context.CreateBatchScopeAsync(CancellationToken.None);
```

Comment:
>CancellationToken.None [](http://example.com/codeflow?start=96&length=22)

Do you remember why we didn't use the cancellationToken available to us here?

Reply:
No, I believe we should pass it. Pushing an update.